Documentation/operations/upgrade.rst

```diff
@@ -318,6 +318,22 @@ Annotations:
   applications create new connections after the TTL specified by the upstream
   DNS server is expired.
 
+New Options
+~~~~~~~~~~~
+
+* ``routing-mode=native``: This option enables native-routing mode, in place of
+  ``tunnel=disabled``, now deprecated.
+* ``tunnel-protocol``: This option allows setting the tunneling protocol, in place
+  of e.g., ``tunnel=vxlan``.
+
+Deprecated Options
+~~~~~~~~~~~~~~~~~~
+
+* The ``tunnel`` option was deprecated and will be removed in v1.14. To enable
```

Comment:
I think it would help to specify the version in which the option was deprecated. For example:

```suggestion
* The ``tunnel`` option was deprecated in v1.1<x> and will be removed in v1.14. To enable
```

Reply:
I'm not sure I understand this one, but maybe the confusion comes from the typo here: this sentence should say "will be removed in v1.15", as per Chris's comment above.

Then, this whole section of the guide is about v1.14. Do we really want to specify that each item in the list is happening in v1.14?

Reply:
> maybe the confusion comes from the typo here: this sentence should say "will be removed in v1.15", as per Chris's comment above.

There's no confusion about the version in which the option will be removed. I saw Chris's feedback before I reviewed. My suggestion was specifically to indicate the version in which the option was deprecated (presumably v1.14).

> Then, this whole section of the guide is about v1.14. Do we really want to specify that each item in the list is happening in v1.14?

If the entire context of the document is explicitly specific to v1.14 and unlikely to be reused in a different, non-specific context, then I think it's fine to ignore my suggestion.

Looking at the file, I see nothing to indicate a specific release in the title or description. Rather, I see that the page is implied to be version-specific because of the branch in which the file will reside when merged. 

The challenge, then, is that when this page is updated in 1.15 and subsequent versions, there will be nothing to indicate the version in which the options were removed--because the version in which deprecation occurred is implied, not explicit.

Does that make sense?

> I keep using past tense when the other entries are using present or present continuous. 

It's not a problem of verb tense agreement; it's a problem of implied vs. explicit. 

> Maybe that would help clarify we're still talking about v1.14?

I certainly think so. 😄

Reply:
(FWIW I agree with @zacharysarah here that we should just be more explicit, but yeah seems fine to just update them all at once in another PR to set that expectation moving forwards)